src/shared_utilities/models.jl

```diff
@@ -418,5 +426,6 @@ function initialize(model::AbstractModel{FT}) where {FT}
     Y = initialize_prognostic(model, coords)
     p = initialize_auxiliary(model, coords)
     p = add_drivers_to_cache(p, model, coords)
+    p = add_parameters_to_cache(p, model)
```

Comment:
I just have a question here - should we store model.parameters in the cache, and then always access it this way (it is also stored in model.parameters, and usually we have accessed it that way in all of our RHS functions)?

Also, should we just store model.parameters.earth_param_set in the cache? Some parameters (e.g. soil spatially varying parameters, which are not in the earth_param_set but are in the model.parameters object) are large - multiple 3D fields. we probably dont want two copies

Reply:
What work comes next on the parameters/what issues should we open? are we moving towards not storing parameters in model.parameters, but rather in p.parameters?

Reply:
I haven't looked at this code for 3 weeks and forgot everything.

I think you are right: we only want the earth_param_set in the cache, and the model parameters being only the parameters for the model (ie, without the earth_param_set)